pkg/controller/restore_controller.go

```diff
@@ -233,20 +236,20 @@ func (c *restoreController) processRestore(key string) error {
 	log.Debug("Running restore")
 
 	// execution & upload of restore
-	restoreWarnings, restoreErrors, restoreFailure := c.runRestore(
+	apiRestoreResults, restoreFailure := c.runRestore(
```

Comment:
A bit of a nit pick :) but I'd rather `apiRestoreResults` be in the singular, so `apiRestoreResult`.

Reply:
I'd prefer not to have `api` as part of the name, if possible